pyproject.toml

```diff
@@ -1,3 +1,48 @@
+[tool.coverage.run]
```

Comment:
Yeah, I think so -- I'm kind of surprised this passed as-is, but maybe it's because the coverage on the 3.11+ jobs (where `[toml]` isn't needed) was sufficient?

Reply:
Probably a bunch of other stuff depends on toml so it's implicitly there.

Reply:
a73bba7..086cecc